support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -94,48 +94,46 @@ export const inThreeTier = threeTierCheckoutEnabled(
 	commonState.amounts,
 );
 
+// Lazy load your components
+// const ThreeTierLanding = lazy(() => import('./twoStepPages/threeTierLanding'));
+const SupporterPlusCheckout = lazy(
+	() => import('./twoStepPages/secondStepCheckout'),
+);
+const SupporterPlusThankYou = lazy(
+	() => import('pages/supporter-plus-thank-you/supporterPlusThankYou'),
+);
+
+console.log('***');
+
 // ----- Render ----- //
 
 const router = () => {
 	return (
 		<BrowserRouter>
 			<ScrollToTop />
 			<Provider store={store}>
-				<Routes>
-					{countryIds.map((countryId) => (
-						<>
-							<Route
-								path={`/${countryId}/contribute/:campaignCode?`}
-								element={
-									/*
-									 * if you are coming to the /contribute route(s) and you also have a one off
-									 * contribution type (set in the url) and find yourself in the three tier
-									 * variant we should redirect you to the /contribute/checkout route
-									 */
-									inThreeTier ? (
-										<ThreeTierRedirectOneOffToCheckout countryId={countryId}>
-											<ThreeTierLanding />
-										</ThreeTierRedirectOneOffToCheckout>
-									) : (
-										<SupporterPlusInitialLandingPage
-											thankYouRoute={thankYouRoute}
-										/>
-									)
-								}
-							/>
-							<Route
-								path={`/${countryId}/contribute/checkout`}
-								element={
-									<SupporterPlusCheckout thankYouRoute={thankYouRoute} />
-								}
-							/>
-							<Route
-								path={`/${countryId}/thankyou`}
-								element={<SupporterPlusThankYou />}
-							/>
-						</>
-					))}
-				</Routes>
+				<Suspense fallback={<ThreeTierLanding />}>
```

Comment:
Presume this is the fallback main page load

Reply:
Yep, I just added that here and didn't do a lot of thinking about tbh, the examples I used had something like `<Suspense fallback={<div>loading....</div>}>`, so you could probably have a loading fallback, which would make sense (be more lightweight) than loading `ThreeTierLanding` as the fallback until the lazily loaded component is ready.